### DISCUSSION TITLE
Generic ESP8285 Full-duplex 2.4GHz RX as TX

### DISCUSSION THREAD

**Discussion by theglasshammer** [ExpressLRS/ExpressLRS#2762] [General]
Hi, how to turn something like [RP1 ](https://www.radiomasterrc.com/products/rp1-expresslrs-2-4ghz-nano-receiver) into a transmitter? I was able to install the "Generic ESP8285 Full-duplex 2.4GHz RX as TX" target on it and it boots, but fails to connect to a receiver (when the same binding phrase is set via web ui). Once you enable the Airport mode, it stops working (booting cleanly) at all. Tested with v3.3.0 all the way up to v3.4.2.

**Comment by CloudedQuartz** [comment]
Hi, were you able to get this working? I'm trying to save my old fs I6x by turning it into a transmitter lol
